SVGLayer.m

```diff
@@ -71,6 +71,7 @@ - (void)setPaths:(NSArray<SVGBezierPath*> *)paths
             continue;
         }        
         CAShapeLayer * const layer = [CAShapeLayer new];
+        layer.contentsScale = [UIScreen mainScreen].scale;
```

Comment:
to ensure this also compiles for macOS targets, so could you please wrap this line in the `#if` as in https://github.com/pocketsvg/PocketSVG/pull/159/files#diff-46bcce9e88f028f08aab10cc7dd73808L23 ?

Reply:
Ah, sure - should have checked Travis